Lib/Common.Split/NetFx/Properties/AssemblyInfo.cs

```diff
@@ -35,9 +35,9 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 
-[assembly: AssemblyVersion("11.1.6.0")]
-[assembly: AssemblyFileVersion("11.1.6.0")]
-[assembly: AssemblyInformationalVersion("11.1.6.0")]
+[assembly: AssemblyVersion("11.2.0.0")]
```

Comment:
I suggest to not bump version number in this PR. Historically, we prepare release in separate commit along with release notes and other release celebration. So this should be done after all changes for new release are in master.

Reply:
Updated.